source/helpers/CaseDataConverter.js

```diff
@@ -62,8 +68,10 @@ export const convertAnswersToArray = (data, formQuestions) => {
 
   Object.entries(data).forEach((answer) => {
     const [fieldId, value] = answer;
-    const { id, type, tags, disableValueStorage, ...other } =
-      formQuestions.find((element) => element.id === fieldId);
+    const match = formQuestions.find((element) => element.id === fieldId);
+
+    const { type, tags, disableValueStorage, ...other } =
```

Comment:
There is a chance that all properties could be undefined, but that is OK or?

Reply:
is OK!